app/assets/stylesheets/application.scss

```diff
@@ -257,6 +257,7 @@ table {
 
 .table-fixed {
   table-layout: fixed;
+  margin: 0px !important;
```

Comment:
`!important` is often a sign that somewhere else a more specific selector has different style set; if possible, it should be overridden at the more specific selector level where it is needed rather then forced with `!important` which will make it almost impossible to override if needed.